static/app/components/version.tsx

```diff
@@ -63,6 +67,8 @@ function Version({
   const versionToDisplay = formatVersion(version, withPackage);
   const theme = useTheme();
 
+  useReleasesDrawer();
```

Comment:
We probably don't want this hook in `Version`, probably better to move it higher up?